### DISCUSSION TITLE
Why no support for 'firebase' web sdk

### DISCUSSION THREAD

**Discussion by zgover** [wovalle/fireorm#294] [Q&A] [answered]
Why no support for 'firebase' web sdk and only support for 'firebase-admin' node.js sdk?

**Comment by wovalle** [comment, marked as answer]
The web sdk is entirely different from firebase-admin and and the original idea was to add support for firebase-admin.

fireorm could be extended to support the web sdk but the effort would be huge and sadly I don't have time for such commitment right now.
